### ISSUE TITLE
Make the default output data `Float32` if using the Copernicus DEM

### ISSUE THREAD

**Issue by scottstanie** [scottstanie/sardem#17]
Even though SRTM is `int16`, there's no reason to force the conversion of the COP DEM to that same data type.

**Comment by xiao-cai-ji0122** [comment]
I have the same question.

**Comment by scottstanie** [comment]
Yes I'm sorry to have broken some users' workflows that were relying on defaults, but I made the change since overall, it was a weird default to have the native Float32 Copernicus tiles get stitched together into an Int16 raster. It only made sense for SRTM data, which is Int16

Thanks for noting the issue for other isce2 users
